docs/app-store-connect/get-latest-build-number.md

```diff
@@ -0,0 +1,102 @@
+
+get-latest-build-number
+=======================
+
+
+**Find the highest build number used for given app considering both TestFlight and App Store submissions**
+### Usage
+```bash
+app-store-connect get-latest-build-number [-h] [--log-stream STREAM] [--no-color] [--version] [-s] [-v]
+    [--log-api-calls]
+    [--api-unauthorized-retries UNAUTHORIZED_REQUEST_RETRIES]
+    [--api-server-error-retries SERVER_ERROR_RETRIES]
+    [--disable-jwt-cache]
+    [--json]
+    [--issuer-id ISSUER_ID]
+    [--key-id KEY_IDENTIFIER]
+    [--private-key PRIVATE_KEY]
+    [--certificates-dir CERTIFICATES_DIRECTORY]
+    [--profiles-dir PROFILES_DIRECTORY]
+    [--platform PLATFORM]
+    APPLICATION_ID_RESOURCE_ID
+```
+### Required arguments for action `get-latest-build-number`
+
+##### `APPLICATION_ID_RESOURCE_ID`
+
+
+Application Apple ID. An automatically generated ID assigned to your app
+### Optional arguments for action `get-latest-build-number`
+
+##### `--platform=IOS | MAC_OS | TV_OS`
+
+
+Apple operating systems
+### Optional arguments for command `app-store-connect`
+
+##### `--log-api-calls`
+
+
+Turn on logging for App Store Connect API HTTP requests
+##### `--api-unauthorized-retries, -r=UNAUTHORIZED_REQUEST_RETRIES`
+
+
+Specify how many times the App Store Connect API request should be retried in case the called request fails due to an authentication error (401 Unauthorized response from the server). In case of the above authentication error, the request is retried usinga new JSON Web Token as many times until the number of retries is exhausted. If not given, the value will be checked from the environment variable `APP_STORE_CONNECT_API_UNAUTHORIZED_RETRIES`. [Default: 3]
```

Comment:
```suggestion
Specify how many times the App Store Connect API request should be retried in case the called request fails due to an authentication error (401 Unauthorized response from the server). In case of the above authentication error, the request is retried using a new JSON Web Token as many times until the number of retries is exhausted. If not given, the value will be checked from the environment variable `APP_STORE_CONNECT_API_UNAUTHORIZED_RETRIES`. [Default: 3]
```

Reply:
Won't change in this PR.